tpm.go

```diff
@@ -356,10 +368,23 @@ var openDefaultTcti = func() (io.ReadWriteCloser, error) {
 func connectToDefaultTPM() (*tpm2.TPMContext, error) {
 	tcti, err := openDefaultTcti()
 	if err != nil {
+		if isPathError(err) {
+			return nil, nil
```

Comment:
returning a nil pointer and no error invites panics,  is there are reason to do that instead of returning the error of the level above already here? I don't see wrapping in the level above that would get confused atm